right-for-life_back/utils/configs.js

```diff
@@ -0,0 +1,14 @@
+const dbName = 'right-for-life';
+const dbOptions = {
+  useUnifiedTopology: true,
+  useNewUrlParser: true,
+  dbName: dbName,
+};
+
+module.exports.dbName = dbName;
+module.exports.dbURL = `mongodb://localhost:27017/${dbName}`;
```

Comment:
Sorry if dumb question, but if we use external Mongo DB, why it's localhost here?

Reply:
> Sorry if dumb question, but if we use external Mongo DB, why it's localhost here?

it didn't used right now:)